SIS_import/utils.rb

```diff
@@ -0,0 +1,3 @@
+def json_parse_nil(json)
+		JSON.parse(json) if json && json.length >= 2
```

Comment:
I'd make it more explicit what is done / returned if the parse doesn't work.  Also its likely you could put the rescue in this method and not have to repeat it in the callers of json_parse_nil.  The name "json_parse_safe" might be better if that is really what this method does.